tools/sst_dump_tool.cc

```diff
@@ -381,6 +493,21 @@ Status SstFileDumper::ReadSequential(bool print_kv, uint64_t read_num,
     }
 
     if (print_kv) {
+      if (options_.pegasus_data) {
+        if (ikey.user_key.size() >= 2) {
+          uint32_t expire_ts = 0;
+          std::string hash_key, sort_key, user_data;
+          pegasus_restore_key(ikey.user_key, hash_key, sort_key);
+          pegasus_restore_value(value, expire_ts, user_data);
+          std::ostringstream oss;
+          oss << "\"" << escape_string(hash_key) << "\" : \"" << escape_string(sort_key)
+              << "\" @ " << ikey.sequence << " : " << ikey.type << " => "
+              << expire_ts << " : \"" << escape_string(user_data) << "\"";
+          fprintf(stdout, "%s\n", oss.str().c_str());
```

Comment:
In pr 23, If it is not pegasus_data should also print some info. Is it useless?
code in pr 23:
```
 if (!options_.pegasus_data) {
        fprintf(stdout, "%s => %s\n",
            ikey.DebugString(output_hex_).c_str(),
            value.ToString(output_hex_).c_str());
      } else {
        if (ikey.user_key.size() >= 2) {
          uint32_t expire_ts = 0;
          std::string hash_key, sort_key, user_data;
          pegasus_restore_key(ikey.user_key, hash_key, sort_key);
          pegasus_restore_value(value, expire_ts, user_data);
          std::ostringstream oss;
          oss << "\"" << escape_string(hash_key) << "\" : \"" << escape_string(sort_key)
              << "\" @ " << ikey.sequence << " : " << ikey.type << " => "
              << expire_ts << " : \"" << escape_string(user_data) << "\"";
          fprintf(stdout, "%s\n", oss.str().c_str());
        }
      }
```

Reply:
> In pr 23, If it is not pegasus_data should also print some info. Is it useless?
> code in pr 23:
> 
> ```
>  if (!options_.pegasus_data) {
>         fprintf(stdout, "%s => %s\n",
>             ikey.DebugString(output_hex_).c_str(),
>             value.ToString(output_hex_).c_str());
>       } else {
>         if (ikey.user_key.size() >= 2) {
>           uint32_t expire_ts = 0;
>           std::string hash_key, sort_key, user_data;
>           pegasus_restore_key(ikey.user_key, hash_key, sort_key);
>           pegasus_restore_value(value, expire_ts, user_data);
>           std::ostringstream oss;
>           oss << "\"" << escape_string(hash_key) << "\" : \"" << escape_string(sort_key)
>               << "\" @ " << ikey.sequence << " : " << ikey.type << " => "
>               << expire_ts << " : \"" << escape_string(user_data) << "\"";
>           fprintf(stdout, "%s\n", oss.str().c_str());
>         }
>       }
> ```

Yes, there some code block changes.
Old code:
```
if (not pegasus_data) {
  do A
} else {
  do B
}
```
Now
```
if (pegasus_data) {
  do B
  continue;
}
do A
```
So, there are no logic modifications, just try to keep code difference as small as possible.